src/Tribe/Views/V2/Hooks.php

```diff
@@ -51,7 +51,6 @@ public function register() {
 	protected function add_actions() {
 		add_action( 'rest_api_init', [ $this, 'register_rest_endpoints' ] );
 		add_action( 'tribe_common_loaded', [ $this, 'on_tribe_common_loaded' ], 1 );
-		add_action( 'loop_start', [ $this, 'on_loop_start' ], PHP_INT_MAX );
 		add_action( 'wp_head', [ $this, 'on_wp_head' ], PHP_INT_MAX );
```

Comment:
Using `PHP_INT_MAX` in a filter is the equivalent of `!important` in CSS; I suggest we use a `999` here.